src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars-list">
+      <li class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </li>
+      <li class="stars stars--1">
```

Comment:
Add empty lines between multiline sibling blocks
Point 1 [`link`](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)

```suggestion
      </li>
      
      <li class="stars stars--1">
```